src/Files.App/Helpers/UI/AppSystemBackdrop.cs

```diff
@@ -54,7 +54,17 @@ protected override void OnTargetDisconnected(ICompositionSupportsSystemBackdrop
 			base.OnTargetDisconnected(disconnectedTarget);
 			this.target = null!;
 			this.root = null!;
-			controller?.RemoveSystemBackdropTarget(disconnectedTarget);
+
+
+			try
+			{
+				controller?.RemoveSystemBackdropTarget(disconnectedTarget);
+			}
+			catch (ObjectDisposedException)
```

Comment:
Do you think we should just catch all kind of exceptions?

Reply:
The truth is I'd rather they throw so that we get notified about the issue.